packages/pds/src/app-view/services/indexing/index.ts

```diff
@@ -110,9 +110,5 @@ export class IndexingService {
     await this.db.db.deleteFrom('profile').where('creator', '=', did).execute()
     await this.db.db.deleteFrom('repost').where('creator', '=', did).execute()
     await this.db.db.deleteFrom('like').where('creator', '=', did).execute()
-    await this.db.db
```

Comment:
oh lol how'd that get in twice 😅

Reply:
Haha _reeallly_ making sure those rows don't stick around 😆